### PR TITLE
change default to cluster v4

### PR DESCRIPTION
**Pull request by bogui56**
[comment]: <> (Please tell us something about this pull request)

## Types of changes
[comment]: <> ( What types of changes does your code introduce? Put an `x` in all the boxes that apply: )
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work for users)
- [ ] Requiring change in macros repository (Please provide links to the macros pull request in the last section)
- [x] I am a member of [GitHub organization of sPHENIX Collaboration](https://github.com/orgs/sPHENIX-Collaboration/people), EIC, or ECCE (contact Chris Pinkenburg to join)

## What kind of change does this PR introduce? (Bug fix, feature, ...)

[comment]: <> ( What does this PR do? Linking to talk in software meeting encouraged )

change default to cluster v4

## TODOs (if applicable)

[comment]: <> ( In case this is a draft PR, e.g. for running checks using Jenkins, please make the pull request as a draft: https://github.blog/2019-02-14-introducing-draft-pull-requests/  )

### CODE REVIEW

**Comment by sPHENIX-GitHub-l** [comment]
Hi Joe, 

I agree. I created this PR to see what the QA stuff says.

there is something else thats wrong. The reference efficinecies look rather bad already. 
And I tried to get a fresh checkout and test things before launching the pull request, but
the checkout didn't compile. 

I got a bunch ACTS errors. See below.

We should try to fix this too somehow. But I am not sure where to start. The jenkins setup seems to work
my local checkout doesn't. Wierd.

I will look at the QA modules.

cheers

   Christof 



PHActsSiliconSeeding.cc: In member function 'Acts::SpacePointGridConfig PHActsSiliconSeeding::configureSPGrid()':
PHActsSiliconSeeding.cc:714:10: error: 'struct Acts::SpacePointGridConfig' has no member named 'phiBinDeflectionCoverage'
   config.phiBinDeflectionCoverage = m_numPhiNeighbors;
          ^~~~~~~~~~~~~~~~~~~~~~~~
In file included from PHActsGSF.cc:1:
PHActsGSF.h:65:50: error: 'GeneralFitterOptions' in 'class ActsExamples::TrackFittingAlgorithm' does not name a type
       const ActsExamples::TrackFittingAlgorithm::GeneralFitterOptions& options);
                                                  ^~~~~~~~~~~~~~~~~~~~
PHActsGSF.cc: In member function 'virtual int PHActsGSF::InitRun(PHCompositeNode*)':
PHActsGSF.cc:59:55: error: 'makeGsfFitterFunction' is not a member of 'ActsExamples::TrackFittingAlgorithm'
   m_fitCfg.fit = ActsExamples::TrackFittingAlgorithm::makeGsfFitterFunction(
                                                       ^~~~~~~~~~~~~~~~~~~~~
PHActsGSF.cc: In member function 'virtual int PHActsGSF::process_event(PHCompositeNode*)':
PHActsGSF.cc:114:44: error: 'GeneralFitterOptions' is not a member of 'ActsExamples::TrackFittingAlgorithm'
       ActsExamples::TrackFittingAlgorithm::GeneralFitterOptions options{
                                            ^~~~~~~~~~~~~~~~~~~~
PHActsGSF.cc:127:48: error: 'options' was not declared in this scope
       auto result = fitTrack(wrappedSls, seed, options);
                                                ^~~~~~~
PHActsGSF.cc:127:48: note: suggested alternative: 'optind'
       auto result = fitTrack(wrappedSls, seed, options);
                                                ^~~~~~~
                                                optind
PHActsGSF.cc: At global scope:
PHActsGSF.cc:365:50: error: 'GeneralFitterOptions' in 'class ActsExamples::TrackFittingAlgorithm' does not name a type
       const ActsExamples::TrackFittingAlgorithm::GeneralFitterOptions& options)
                                                  ^~~~~~~~~~~~~~~~~~~~
PHActsGSF.cc: In member function 'ActsExamples::TrackFittingAlgorithm::TrackFitterResult PHActsGSF::fitTrack(const std::vector<std::reference_wrapper<const ActsExamples::IndexSourceLink> >&, const TrackParameters&, const int&)':
PHActsGSF.cc:367:52: error: no match for call to '(std::__shared_ptr_access<ActsExamples::TrackFittingAlgorithm::TrackFitterFunction, (__gnu_cxx::_Lock_policy)2, false, false>::element_type {aka ActsExamples::TrackFittingAlgorithm::TrackFitterFunction})

**Comment by osbornjd** [comment]
Merge with master, that will fix your Acts issues. We updated versions yesterday (or resource your environment, maybe you are working with the old Acts build still) .

> there is something else thats wrong. The reference efficinecies look rather bad already.

Depends on what you are remembering as the efficiency. Recently we switched to only accepting tracks that have a silicon match, which gives us a slight knock in efficiency (whereas before TPC only tracks were included in our efficiency, but we decided we likely won't be able to do much with those).

**Comment by osbornjd** [comment]
Yes, I mostly only track the PYTHIA-D0 QA and the low occupancy QA. The high occupancy QA is not representative of hijing events.

**Comment by osbornjd** [comment]
@bogui56 could you fix the TPC z pulls and cluster error in the QA? Probably just the QA modules not pulling the data from the correct cluster version. 
Screen Shot 2022-07-25 at 11 16 49 AM
.

**Comment by osbornjd** [comment]
> @bogui56 could you fix the TPC z pulls and cluster error in the QA? Probably just the QA modules not pulling the data from the correct cluster version. Screen Shot 2022-07-25 at 11 16 49 AM .

@bogui56  Actually this is from a previous issue for which I have submitted a PR for. I am now confused as this PR doesn't actually change the cluster versions, it only changes (apparently) a significant portion of `PHActsSiliconSeeding`. What does this PR actually change?

**Comment by sPHENIX-GitHub-l** [comment]
Hi, 

it looks like the last commit somehow reverted the defaults back to v3. Grrr. this should not have happened. 
This should only have changed the ActsSilliconSeeds code that had changed while I was wokring on the cluster version 
switch. I didn't want to erase those changes. Looks like this royally messed things up. 

Let me try again. Could you send me a pointer to the latest greatest SiliconSeeding code so I can make sure
I have the right version?

Thanks 

   Christof

**Comment by sPHENIX-GitHub-l** [comment]
Ah. Ok. looks like I have been logged in in that session too long. Let me restart.

Have a look at the ref here 35% at high pt doesn't look like what I expected.
With the error parametrization its gets better at least...
